prototype_source/quantization_in_pytorch_2_0_export_tutorial.rst

```diff
@@ -0,0 +1,350 @@
+(prototype) Quantization in PyTorch 2.0 Export Tutorial
+==============================================================
+
+Today we have `FX Graph Mode
+Quantization <https://pytorch.org/docs/stable/quantization.html#prototype-fx-graph-mode-quantization>`__
+which uses ``symbolic_trace`` to capture the model into a graph, and then
+perform quantization transformations on top of the captured model. In a
+similar way, for Quantization 2.0 flow, we will now use the PT2 Export
+workflow to capture the model into a graph, and perform quantization
+transformations on top of the ATen dialect graph. This approach is expected to
+have significantly higher model coverage, better programmability, and
+a simplified UX.
+
+Prerequisites:
+-----------------------
+
+-  `Understanding of torchdynamo concepts in PyTorch <https://pytorch.org/docs/stable/dynamo/index.html>`__
+-  `Understanding of the quantization concepts in PyTorch <https://pytorch.org/docs/master/quantization.html#quantization-api-summary>`__
+-  `Understanding of FX Graph Mode post training static quantization <https://pytorch.org/tutorials/prototype/fx_graph_mode_ptq_static.html>`__
+-  `Understanding of BackendConfig in PyTorch Quantization FX Graph Mode <https://pytorch.org/tutorials/prototype/backend_config_tutorial.html?highlight=backend>`__
+-  `Understanding of QConfigMapping in PyTorch Quantization FX Graph Mode <https://pytorch.org/tutorials/prototype/backend_config_tutorial.html#set-up-qconfigmapping-that-satisfies-the-backend-constraints>`__
+
+Previously in ``FX Graph Mode Quantization`` we were using ``QConfigMapping`` for users to specify how the model to be quantized
+and ``BackendConfig`` to specify the supported ways of quantization in their backend.
+This API covers most use cases relatively well, but the main problem is that this API is not fully extensible
```

Comment:
@jerryzh168 changes to new API is not so much about whether it covers the cases well or not, it is just a new way of thinking of the quantizaiton workflow in the context of PT2 export. So I dont think it is quite accurate to say that we built new workflow to cover tiny percentage of cases that werent working well.

THis is also the reason why i dont want to jump into documenting something that is not quite true to the reason of why new API exist. cc: @supriyar

Reply:
I feel maybe we should mention (3) and (4) as well in https://docs.google.com/document/d/1_jjXrdaPbkmy7Fzmo35-r1GnNKL7anYoAnqozjyY-XI/edit#heading=h.jtqauapwj95c which are not limitations but more talking about motivations for the new API to have the complete picture?

Reply:
Thanks for the suggestions. section 3, 4 are added.

Reply:
I don't think we should be documenting the details on the motivation for quantizer and annotation API to this level in this tutorial. Summarizing the motivation in 1-2 sentences should be good and the details can be left to the appendix or some other detailed doumentation/notes.